modules/aleph/src/Aleph/AlephClient.php

```diff
@@ -236,27 +245,67 @@ public function getDebts(AlephPatron $patron) {
   }
 
   /**
+   * Get the items for a Material.
+   *
    * @param \Drupal\aleph\Aleph\Entity\AlephMaterial $material
    *    The Aleph material to get items from.
    *
-   * @return \SimpleXMLElement The SimpleXMLElement response from Aleph.
+   * @return \SimpleXMLElement
    *    The SimpleXMLElement response from Aleph.
    *
    * @throws AlephClientException
    */
   public function getItems(AlephMaterial $material) {
     return $this->requestRest(
       'GET',
-      'record/' . $this->mainLibrary . $material->getId() . '/items?view=full'
+      'record/' . $this->catalogLibrary . $material->getId() . '/items?view=full'
     );
   }
 
+  /**
+   * Create payment.
+   *
+   * @param \Drupal\aleph\Aleph\Entity\AlephPatron $patron
+   *   The Aleph patron.
+   * @param int $sum
+   *   Amount paid.
+   * @param string $reference
+   *   Identification of the payment.
+   *
+   * @return \SimpleXMLElement|false
+   *   The SimpleXMLElement from the raw XML response.
```

Comment:
This seems to always return a boolean - never a `\SimpleXMLElement`. I am unsure whether the parsing of the XML should actually go in the handler - not in the Client as it is the case here. That would be consistent with the other function calls.